mythril/laser/ethereum/transaction/concolic.py

```diff
@@ -10,13 +10,14 @@ def execute_message_call(laser_evm, callee_address, caller_address, origin_addre
     open_states = laser_evm.open_states[:]
     del laser_evm.open_states[:]
 
+    next_transaction_id = get_next_transaction_id()
```

Comment:
Each transaction should have a unique transaction id, so this should be moved up to the for loop